wemake_python_styleguide/violations/consistency.py

```diff
@@ -1196,5 +1198,38 @@ class UselessOperatorsViolation(ASTViolation):
 
     """
 
-    code = 330
     error_template = 'Found unnecessary operator: {0}'
+    code = 330
+
+
+@final
+class NumberWithMeaninglessZeroViolation(TokenizeViolation):
+    """
+    Enforce consistent octal, hex, and binary numbers.
+
+    Reasoning:
+        Numbers should be consistent.
```

Comment:
Reasoning needs to be improved. Please, add more information about the root cause: readability. We need consistency to make our code readable. And equal numbers might be written in almost infinite ways.